e220/e220.go

```diff
@@ -364,3 +373,20 @@ func (d *Device) ReadByte() (byte, error) {
 	}
 	return data, nil
 }
+
+func (d *Device) Read(p []byte) (n int, err error) {
+	if len(p) == 0 {
+		return 0, nil
+	}
+
+	size := d.uart.Buffered()
+	for size == 0 {
+		runtime.Gosched()
+		size = d.uart.Buffered()
+	}
+
+	if size > len(p) {
+		size = len(p)
+	}
+	return d.uart.Read(p)
```

Comment:
サイズチェックが必要かな。
`Buffered()` と `len(p)` の小さいほうを設定する必要がある。

```suggestion
	return d.uart.Write(p[:size])
```

Reply:
以下で `size` を用いて `uart.Read()` するように変更しました。
7567abb